tests/unit/test_customer_gauge.py

```diff
@@ -0,0 +1,116 @@
+"""'test_customer_gauge.py'."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from viadot.exceptions import APIError, CredentialError
+from viadot.sources.customer_gauge import CustomerGauge, CustomerGaugeCredentials
+
+variables = {
+    "credentials": {
+        "client_id": "client_id",
+        "client_secret": "client_secret",
+    }
+}
+
+
+@pytest.mark.basic()
+def test_customer_gauge_credentials():
+    """Test Customer Gauge credentials."""
+    CustomerGaugeCredentials(
+        client_id=variables["credentials"]["client_id"],
+        client_secret=variables["credentials"]["client_secret"],
+    )
+
+
+@pytest.mark.basic()
+def test_missing_credentials(mocker):
+    """Test raise error without Customer Gauge credentials."""
+    mocker.patch(
+        "viadot.sources.customer_gauge.get_source_credentials", return_value=None
+    )
```

Comment:
Why do you need to patch this? The test should work without that.

Reply:
removed.